### DISCUSSION TITLE
How do you stream a unix socket?

### DISCUSSION THREAD

**Discussion by mbround18** [rwf2/Rocket#2090] [Questions]
I have been looking at [this older doc for v0.4](https://rocket.rs/v0.4/guide/responses/#streaming) which unfortunately does not work in v0.5. How do you get a unix socket to stream to an event stream? 

My end goal is to stream data from a unix socket to an endpoint on a web url.

**Comment by somehowchris** [comment]
https://rocket.rs/v0.5-rc/guide/responses/#async-streams
https://docs.rs/tokio/latest/tokio/net/struct.UnixStream.html

**Reply by mbround18** [reply]
Any chance yall got a better example? I have been struggling with those two docs for a couple days now with no luck.
